lib/castle/bucket/daily.ex

```diff
@@ -12,7 +12,7 @@ defmodule Castle.Bucket.Daily do
   end
 
   def ceiling(time) do
-    if Timex.compare(floor(time), time) == 0 do
+    if Timex.compare(__MODULE__.floor(time), time) == 0 do
```

Comment:
This is an upcoming deprecation (accidentally upgraded my homebrew elixir `1.7.3` to `1.8.2`).  You get a bunch of compilation errors that this module's `floor` function conflicts with the built-in `Kernel.floor`.